sdk/go/pulumi/run.go

```diff
@@ -81,7 +80,12 @@ func RunErr(body RunFunc, opts ...RunOption) error {
 	}
 	defer contract.IgnoreClose(ctx)
 
-	return RunWithContext(ctx, body)
+	err = RunWithContext(ctx, body)
+	if err != nil {
+		err := ctx.Log.Error(fmt.Sprintf("program failed: %v\n", err), nil)
+		contract.AssertNoError(err)
+	}
+	return err
```

Comment:
Handle error logging in the SDK

Reply:
I'd like to add a common looking pulumi error text `an unhandled error occurred:` and move the error message
down a line